code/game/verbs.dm

```diff
@@ -146,7 +146,8 @@
 	if(!islist(old_verbs))
 		old_verbs = list(old_verbs)
 	var/list/removed_verbs = list()
-	for(var/procpath/P as() in old_verbs)
+	for(var/pp in old_verbs)
+		var/procpath/P
```

Comment:
🤔

Reply:
Thats how it is done here, and it works fine 😂 
![image](https://user-images.githubusercontent.com/26465327/127190885-0ab8f292-a3fc-44a1-833b-c082371aab79.png)

Reply:
You didn't set P to anything dweeb